### PR TITLE
add notebook

### PR DESCRIPTION
**Pull request by Omar-Oz**

### CODE REVIEW

File: examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb [hunk 1]
```diff
@@ -0,0 +1,1604 @@
+{
```
Comment (slemasne) [inline examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb:1]: <p>Use the same heading format as other notebooks.</p><p>Example:</p><p> </p><p><code>1. Set up API</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/420/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (slemasne) [inline examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb:1]: <p>Use the format properties param here to make output look nicer </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/420/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Comment (slemasne) [inline examples/use-cases/ibor/Get Holdings and Extract to csv.ipynb:1]: <p>Remove blank cell at end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/finbourne/sample-notebooks/pull/420/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

**Comment by rizwansaeed** [comment]
doesnt this duplicate https://github.com/finbourne/sample-notebooks/blob/master/examples/features/core-lusid/Generating%20an%20IBOR%20extract%20with%20LUSID's%20GetHoldings%20method.ipynb

**Comment by Omar-Oz** [comment]
> doesnt this duplicate https://github.com/finbourne/sample-notebooks/blob/master/examples/features/core-lusid/Generating%20an%20IBOR%20extract%20with%20LUSID's%20GetHoldings%20method.ipynb

Yes it sort of does, although this workflow is based on portfolio holdings that already exist without transactions. We want to
highlight how simple it is to extract

**Pull request merged**